plugin.program.AML/resources/manuals.py

```diff
@@ -27,8 +27,8 @@
 
 # --- Load pdfrw module ---
 import sys
-__addon_id__ = xbmcaddon.Addon().getAddonInfo('id')
-sys.path.insert(0, FileName('special://home/addons').pjoin(__addon_id__).pjoin('pdfrw').getPath())
+addon_id = xbmcaddon.Addon().getAddonInfo('id')
+sys.path.insert(0, FileName('special://home/addons').pjoin(addon_id).pjoin('pdfrw').getPath())
```

Comment:
this is probably not needed. Why is `pdfrw` not moved to a script module and imported by this addon a dependency?